index.js

```diff
@@ -235,13 +235,15 @@ function css(css, file) {
               chalk`{green Finished {bold ${relativePath}} in {bold ${prettyTime}}}`
             )
 
-            console.warn(
-              reporter(
-                Object.assign({}, result, {
-                  messages: result.warnings()
-                })
+            if (result.messages.length) {
```

Comment:
Shouldn't this be `result.warnings().length`? Otherwise, if there's non-warning messages, empty lines will still be printed.

While we're at it, might as well assign `result.warnings()` to a `const` so we don't have to make the call twice.

Reply:
You know the codebase better, I thought to just check for warnings but I thought that #256 changed this on purpose to include all messages.